rpcs3/Emu/Io/usio.cpp

```diff
@@ -10,14 +10,14 @@
 
 LOG_CHANNEL(usio_log);
 
-struct last_game_status
+struct memory
```

Comment:
Please use more "unique" name, or alternatively put it into an anonymous namespace.

Reply:
Changed it to "usio_memory".